test/NVQPP/test-6.cpp

```diff
@@ -15,9 +15,9 @@
 __qpu__ void cccx_measure_cleanup() {
   cudaq::qreg qubits(4);
   // Initialize
+  x(qubits[0]);
```

Comment:
What's the reason for this change? It looks like a decent test to me to check that everything works even when the qubits are not used in the order they are defined.

Reply:
edit: change is needed to match the expected output - test is an xfail for other reasons.